mypy/plugins/attrs.py

```diff
@@ -559,7 +559,7 @@ def _parse_converter(ctx: 'mypy.plugin.ClassDefContext',
 
         # Signal that we have an unsupported converter.
         ctx.api.fail(
-            "Unsupported converter, only named functions and types are currently supported",
```

Comment:
Could we go witih "Unsupported converter; only types and annotated named functions are currently supported"